modules/appeals_api/spec/requests/schemas/shared_schemas_controller_spec.rb

```diff
@@ -36,7 +36,7 @@ def base_path(appeal_type, schema_type)
     end
 
     describe "schema type 'date'" do
```

Comment:
Since this isn't a shared schema anymore, just a basic definition, I think we can remove this particular spec.